README.md

```diff
@@ -596,6 +596,18 @@ $role->givePermissionTo('edit articles');
 $role->revokePermissionTo('edit articles');
 ```
 
+### DecryptException
+
+Working with a lot of permissions, your cache can grow quickly. When using the database cache driver, you might get a `DecryptException`. In that case, just change your cache migration file from:
+
+```php
+$table->text('value');
+```
+to:
+```php
+`$table->longText('value');
```

Comment:
That's right indeed. But maybe nice to have it here in the troubleshooting-section, because this package is also using Laravel's cache system.